io_scene_nif/nif_import.py

```diff
@@ -88,8 +88,8 @@ def execute(self):
             # check that one armature is selected in 'import geometry + parent
             # to armature' mode
             if NifOp.props.skeleton == "GEOMETRY_ONLY":
-                if len(NifCommon.SELECTED_OBJECTS) != 1 or NifCommon.SELECTED_OBJECTS[0].type != 'ARMATURE':
-                    raise util_math.NifError("You must select exactly one armature in 'Import Geometry Only + Parent To Selected Armature' mode.")
+                if len(self.SELECTED_OBJECTS) != 1 or self.SELECTED_OBJECTS[0].type != 'ARMATURE':
```

Comment:
Uould this be affected by operators that don't use a context copy or that are setting objects selected?

Reply:
Possibly, but as we do this first, it probably doesn't matter.